integration_tests/helpers/http_methods_helpers.py

```diff
@@ -14,11 +14,15 @@ def check_response(response: requests.Response, expected_status_code: int):
 
 
 def get(
```

Comment:
@AntoineRR , one question. What does `should_check_response` do?

Can we add a docstring here?

Also, do you think we should keep the `http_method_helpers` homogeneous in terms of the function definition?

Reply:
Thank you :D :D